CodeGeneration/Sources/SyntaxSupport/AttributeNodes.swift

```diff
@@ -74,6 +74,11 @@ public let ATTRIBUTE_NODES: [Node] = [
             // Special arguments for keyword decl name e.g. 'subscript(_:)', and availability arguments.
             kind: .node(kind: .specializeAttributeArgumentList)
           ),
+          Child(
+            name: "specializedArguments",
+            // Special arguments for keyword decl name e.g. 'subscript(_:)', and availability arguments.
```

Comment:
This comment is not correct at this point. More like:
```suggestion
            // Special arguments for generic where clause.
```